lock-api/src/main/java/com/palantir/lock/watch/NoOpLockWatchEventCache.java

```diff
@@ -19,15 +19,13 @@
 import java.util.Collection;
 import java.util.Optional;
 import java.util.Set;
-import java.util.UUID;
-
-import com.google.common.collect.ImmutableSet;
 
 @SuppressWarnings("FinalClass") // mocks
 public class NoOpLockWatchEventCache implements LockWatchEventCache {
     public static final LockWatchEventCache INSTANCE = new NoOpLockWatchEventCache();
-    private static final TransactionsLockWatchEvents NONE = TransactionsLockWatchEvents.failure(
-            LockWatchStateUpdate.snapshot(UUID.randomUUID(), -1L, ImmutableSet.of(), ImmutableSet.of()));
+    private static final TransactionsLockWatchEvents NONE = ImmutableTransactionsLockWatchEvents.builder()
+            .clearCache(true)
```

Comment:
I think this needs a fix, for in-memory: you need to put dummy values into startTsToSequence for the passed transactions. Otherwise, atlasdb-proxy will start throwing.

Reply:
Good catch